### ISSUE TITLE
what's the meaning of the icon

### ISSUE THREAD

**Issue by fangsaihua** [pgbovine/opt-cpp-backend#63]
when i visualize c++ code execution, what's the meaning of the icon in the pointer, such as the shit and the skull.

**Comment by pgbovine** [comment]
yes! one means unallocated memory, and one means uninitialized values (junk values). i should document this better sometime :)
